lib/kinetic_cafe/error_dsl.rb

```diff
@@ -91,9 +94,9 @@ def define_error(options)
         fail ArgumentError, ":key conflicts with class:#{klass}" if options.key?(:key)
 
         key = if status.kind_of?(Symbol) || status.kind_of?(String)
-                "#{klass}_#{KineticCafe::ErrorDSL.namify(status)}"
+                String.new("#{klass}_#{KineticCafe::ErrorDSL.namify(status)}")
```

Comment:
Are we modifying `key` anywhere after this? If not, then I don’t think that `String.new` is necessary here, but this could even be dealt with as:

```
  key = if …
    …
  else
    …
  end

  key = String.new(key)
```